pr_agent/algo/pr_processing.py

```diff
@@ -21,6 +21,7 @@
 
 OUTPUT_BUFFER_TOKENS_SOFT_THRESHOLD = 1000
 OUTPUT_BUFFER_TOKENS_HARD_THRESHOLD = 600
+PATCH_EXTRA_LINES = 3
 
 def get_pr_diff(git_provider: GitProvider, token_handler: TokenHandler, model: str,
                 add_line_numbers_to_hunks: bool = False, disable_extra_lines: bool = False) -> str:
```

Comment:
**Suggestion**: The addition of 'PATCH_EXTRA_LINES' constant is good for configurability. Consider adding a comment to explain its purpose and impact on PR processing. <br> **Code Suggestion**: 
 ```
 +PATCH_EXTRA_LINES = 3 # Number of extra lines to include in patches for context
 ```